### PR TITLE
Implemented base interfaces for the Task service.

### PR DESCRIPTION
**Pull request by ZhakalenDk**
**Note:** No logic included

### CODE REVIEW

File: ITGuru.FourWheels/ITGuru.FourWheels.Service/Extensions/ITaskExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+﻿using ITGuru.FourWheels.Data.DataModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ITGuru.FourWheels.Service
```
Comment (jeongoks) [inline ITGuru.FourWheels/ITGuru.FourWheels.Service/Extensions/ITaskExtensions.cs:8]: Is it on purpose that this it commented ?

**Review by jeongoks — COMMENTED** [review]
See comment in Files changed.

File: ITGuru.FourWheels/ITGuru.FourWheels.Service/Extensions/ITaskExtensions.cs (hunk 1 above)
Comment (ZhakalenDk) [reply ITGuru.FourWheels/ITGuru.FourWheels.Service/Extensions/ITaskExtensions.cs:8]: Yes. YOu'll not have the need for it, however, I need the systems from `Data` to actually form the logic.

Comment (jeongoks) [reply ITGuru.FourWheels/ITGuru.FourWheels.Service/Extensions/ITaskExtensions.cs:8]: Got it, i'll approve! c:

**Review by jeongoks — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Looks good to me 👍

**Pull request merged**